### PR TITLE
Fix the support of generating Example Model of DataFactoryKeyVaultReference type of DataFactoryElement

### PR DESCRIPTION
**Pull request by RodgeFu**
Fix the support of generating Example Model of DataFactoryKeyVaultReference type of DataFactoryElement according to the change in Azure.Core.Expression.DataFactory

### CODE REVIEW

File: tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts [hunk 1]
```diff
@@ -231,17 +231,30 @@ export class ExampleValue {
         return instance;
     }
 
-    private static createSchemaForDfeObject(raw: any, dfeFormat: string): ObjectSchema | undefined {
+    private static createSchemaForDfeObject(session: Session<TestCodeModel>, raw: any, dfeFormat: string): ObjectSchema | undefined {
```
Comment (raych1) [inline tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts:234]: Will this breaking change break other scenarios?

**Comment by raych1** [comment]
Hi @RodgeFu , which version of this package is used in dotnet sample generation? I heard from the version after 2.5.0(included) with webpack change never works.

File: tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts (hunk 1 above)
Comment (RodgeFu) [reply tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts:234]: no, this method is added in my previous check-in and not used in other scenarios.

**Comment by RodgeFu** [comment]
> Hi @RodgeFu , which version of this package is used in dotnet sample generation? I heard from the version after 2.5.0(included) with webpack change fails to compile due to dependencies resolve error.

I think I updated it to use 2.6 a few weeks ago and didn't see any error in build and publish.

**Review by ArthurMa1978 — APPROVED** [review]

**Pull request merged**